gravitee-apim-console-webui/src/management/api/policy-studio/debug/components/policy-studio-debug-inspector/policy-studio-debug-inspector.component.ts

```diff
@@ -14,11 +14,138 @@
  * limitations under the License.
  */
 
-import { Component } from '@angular/core';
+import { AfterViewInit, Component, Input, OnChanges, SimpleChanges } from '@angular/core';
+import { FlatTreeControl } from '@angular/cdk/tree';
+import { MatTreeFlatDataSource, MatTreeFlattener } from '@angular/material/tree';
+
+import { RequestDebugStep, ResponseDebugStep } from '../../models/DebugStep';
+
+interface Node {
+  name?: string;
+  children?: Node[];
+  type?: string;
+  input?: any;
+  output?: any;
+}
+
+interface FlatNode {
+  expandable: boolean;
+  name: string;
+  level: number;
+}
+
+const httpProperties: Record<string, any> = {
```

Comment:
We could create an interface with `name` & `type` and use it here instead of `any`